frontend/components/Company.vue

```diff
@@ -72,7 +72,7 @@
                     class="text-xs md:text-sm text-gray-900"
                     style="color: #000000; font-family: 'Inter', sans-serif"
                   >
-                    {{ company.company_name }}
+                    A local company
```

Comment:
This should be something like this `{{ company.company_name || "A local company" }}`

Reply:
![Screenshot from 2022-04-16 09-14-57](https://user-images.githubusercontent.com/34966088/163667826-3f52a44b-6849-4f5e-bd1e-e55dbb13696a.png)